### PR TITLE
minor fix for typo, link

### PR DESCRIPTION
**Pull request by mustfaunlu**

### CODE REVIEW

File: docs/topics/inline-functions.md [hunk 1]
```diff
@@ -180,13 +180,13 @@ fun main(s: Array<String>) {
 }
 ```
 
-Normal functions (not marked as inline) cannot have reified parameters.
+Normal functions (not marked as inline) cannot have `reified` parameters.
```
Comment (sarahhaggarty) [inline docs/topics/inline-functions.md:183]:
```suggestion
Normal functions (not marked as inline) cannot have reified parameters.
```

File: docs/topics/inline-functions.md [hunk 2]
```diff
@@ -180,13 +180,13 @@ fun main(s: Array<String>) {
...
 A type that does not have a run-time representation (for example, a non-reified type parameter or a fictitious type like
-`Nothing`) cannot be used as an argument for a reified type parameter.
+`Nothing`) cannot be used as an argument for a `reified` type parameter.
```
Comment (sarahhaggarty) [inline docs/topics/inline-functions.md:185]:
```suggestion
`Nothing`) cannot be used as an argument for a reified type parameter.
```

**Review by sarahhaggarty — CHANGES_REQUESTED** [review]
Thanks for your suggestion @mustfaunlu! We only format "reified" as code when referring to the modifier so I have reverted these changes. I think the addition of the link is a great idea though! If you can take a look at my suggested changes, we can proceed to merge.

**Comment by mustfaunlu** [comment]
Thank you for the clarification, @sarahhaggarty!

**Review by koshachy — APPROVED** [review]

**Review by sarahhaggarty — APPROVED** [review]

**Pull request merged**